datablock.py

```diff
@@ -1502,7 +1501,7 @@ def from_dict(obj, check_format=True, directory=None):
     def from_json(string, check_format=True, directory=None):
         """Decode a datablock from JSON string."""
         return DataBlockFactory.from_dict(
-            json.loads(string, object_hook=_decode_dict),
+            json.loads(string),
```

Comment:
was the `object_hook` stuff really useless?

Reply:
on Python 3 it was a no-op